meow/frontend/src/components/EditPost/EditForm.js

```diff
@@ -24,7 +24,33 @@ const formItemLayout = {
   }
 };
 
+let TWITTER_MAX_LENGTH = 232; //this is hardcoded and does not change if the backend changes.
+
 class EditForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      twitter_length: 0 //even if post_twitter has something, the constructor will be called
+      //before that data is avialable so just set it to 0.
+    };
+  }
+
+  static getDerivedStateFromProps(props, current_state) {
```

Comment:
Is this the right way to do this @dustinnewman98?

Reply:
I think so? This is because the `EditForm` depends on changes in the props right? The value of the tweet comes from the `Form` not the `EditForm`?